Documentation/ProjectFile/gml/surfaces/surface/element/i_element.md

```diff
@@ -1 +1,3 @@
-\ogs_missing_documentation
+Defines a triangle by its three vertices.
+The face normal of the triangle is determined such that the vertices are
+located counter-clockwise around it (right-hand rule).
```

Comment:
Question to reviewers: Does the right-hand rule thing hold true for all of our code?